app/src/main/res/values/styles.xml

```diff
@@ -246,6 +249,7 @@
         <item name="android:maxLines">1</item>
         <item name="android:background">@android:color/transparent</item>
         <item name="android:textCursorDrawable">@null</item>
+        <item name="android:textColorHighlight">@color/drawer_item_highlighting_pressed</item>
```

Comment:
The drawer is not used anymore, but this color is ok for highlighting. In future we should rename this color.